docs/guides/c-extension-for-python.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Extending Qiskit in Python with C
+description: How to build a C extension for your Qiskit workflow in Python
+---
+
+# A little C with your Python
+
+To accelerate your Qiskit Python programs with C, we can use Qiskit's C extension for Python.
+This requires additional steps to the standalone C usage, see the [respective guide](./install-c-api).
+
+<Admonition type="note">
+  These instructions have only been tested on unix-based systems. Windows instructions are in progress._
```

Comment:
```suggestion
  These instructions have only been tested on UNIX-based systems. Windows instructions are in progress._
```

Reply:
I should've clarified this: it's only an internal comment -- the final version that we publish should have tested windows instructions too 🙂 but we can also keep this comment as if it were to be published and remove it once we add windows instructions